packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -178,12 +181,12 @@ describeNoCompat(`Dehydrate Rehydrate Container Test`, (getTestObjectProvider) =
         it("Dehydrated container snapshot 2 times with changes in between", async () => {
             const { container, defaultDataStore } =
                 await createDetachedContainerAndGetRootDataStore();
-            const snapshotTree1: ISnapshotTree = JSON.parse(container.serialize());
+            const snapshotTree1: ISnapshotTree = convertDetachedContainerSnapshot(JSON.parse(container.serialize()));
```

Comment:
Looking at the usage, the name does not tell me really what it does.
getSnapsthotTreeFromSerializedContainer ?